src/Symfony/Component/HttpKernel/Attribute/AsController.php

```diff
@@ -12,7 +12,11 @@
 namespace Symfony\Component\HttpKernel\Attribute;
 
 /**
- * Service tag to autoconfigure controllers.
+ * To autoconfigure your controllers as serivces you can use 
```

Comment:
Thanks @muchafm. I like the improvement ... but I'd reword it and expand it a bit. For example:

```
 * Autoconfigures controllers as services by applying the `controller.service_arguments`
 * tag to them. This allows to inject services in controller action methods, not
 * only in their constructors.
```

Reply:
Oh I see, thank you for the suggestion!